Userland/Libraries/LibELF/DynamicLoader.h

```diff
@@ -152,7 +153,7 @@ class DynamicLoader : public RefCounted<DynamicLoader> {
     VirtualAddress m_dynamic_section_address;
 
     size_t m_tls_offset { 0 };
-    size_t m_tls_size { 0 };
+    size_t m_tls_size { 0 }; // TLS size of the current object
```

Comment:
Instead of putting a comment here, where nobody will see it, let's instead call the variable `m_tls_size_of_current_object` :)